### DISCUSSION TITLE
control through another bot

### DISCUSSION THREAD

**Discussion by ElerWohlmuthFH** [jagrosh/MusicBot#637] [Ideas]
Hello,

I have another Discord-bot that communicates with my Minecraft-server. What it does is it basically sends messages from my Minecraft server to my Discord. Now it would be cool, to control the MusicBot from the Minecraft chat. To do so, it would need a feature to not ignore other bots and to maybe let the MusicBot join a dedicated channel with a dedicated channel ID if the command comes from another bot. Would that be a feature you'd be interested in adding?

**Comment by jagrosh** [comment]
This won't be added, it is poor practice for bots to respond to other bots. If you're trying to set something like this up, you should write your own bot that does both parts.

**Reply by ElerWohlmuthFH** [reply]
Thanks for the answer.
I'm already on it!
